doc/model/generate-svgs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
I think you should use ```#!/bin/bash``` here because ```set -e``` and ```set -x``` are bash features, but not 100% sure.